src/aliases/api/handlers/handlers.go

```diff
@@ -0,0 +1,166 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	jsonutils "github.com/consensys/quorum-key-manager/pkg/json"
+	"github.com/consensys/quorum-key-manager/src/aliases"
+	"github.com/consensys/quorum-key-manager/src/aliases/api/types"
+	aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
+	"github.com/gorilla/mux"
+)
+
+type AliasHandler struct {
+	alias aliases.Alias
+}
+
+func NewAliasHandler(backend aliases.Alias) *AliasHandler {
+	h := AliasHandler{
+		alias: backend,
+	}
+
+	return &h
+}
+
+func (h *AliasHandler) Register(r *mux.Router) {
+	regRoute := r.PathPrefix("/registries/{registry_name}").Subrouter()
+	regRoute.HandleFunc("", h.deleteRegistry).Methods(http.MethodDelete)
+
+	alRoute := regRoute.PathPrefix("/aliases").Subrouter()
+	alRoute.HandleFunc("", h.createAlias).Methods(http.MethodPost)
+	alRoute.HandleFunc("", h.listAliases).Methods(http.MethodGet)
+	alRoute.HandleFunc("/{alias_key}", h.getAlias).Methods(http.MethodGet)
+	alRoute.HandleFunc("/{alias_key}", h.updateAlias).Methods(http.MethodPut)
+	alRoute.HandleFunc("/{alias_key}", h.deleteAlias).Methods(http.MethodDelete)
+}
+
+func (h *AliasHandler) deleteRegistry(w http.ResponseWriter, r *http.Request) {
```

Comment:
Please, remember to include endpoints annotations docs as you can see in `/stores/api/handlers/eth1.go` 
```
// @Summary Creates a secret
// @Description Create new secret on selected Store
// @Tags Secrets
// @Accept json
// @Produce json
// @Param id path string true "Secret Identifier"
// @Param storeName path string true "Store Identifier"
// @Param request body types.SetSecretRequest true "Create Secret request"
// @Success 200 {object} types.SecretResponse "Secret data"
// @Failure 400 {object} ErrorResponse "Invalid request format"
// @Failure 404 {object} ErrorResponse "Store not found"
// @Failure 500 {object} ErrorResponse "Internal server error"
// @Router /stores/{storeName}/secrets/{id} [post]
```

Reply:
Completely forgot about that.